pkg/uefi/meregion.go

```diff
@@ -0,0 +1,238 @@
+// Copyright 2018 the LinuxBoot Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package uefi
+
+import (
+	"bytes"
+	"encoding/binary"
+	"encoding/hex"
+	"fmt"
+	"log"
+)
+
+// ME Partition parsing, the goal is to spot a padding in the ME Region
+// after the ME partitions so that this region can be shrunk in the IFD.
+//
+// ME partition informations from http://me.bios.io/ME_blob_format
```

Comment:
Place this comment directly in front of "package uefi". This will give the nicest output from running "go doc".

Reply:
right thanks, I rarely use go doc... I should check it more

Reply:
except `golint` disagrees:  pkg/uefi/meregion.go:5:1: package comment should be of the form "Package uefi ..."

Reply:
Oh right, I forgot all of uefi is in one package.